src/main/java/com/ibm/etcd/client/EtcdClient.java

```diff
@@ -580,7 +585,7 @@ public boolean isClosed() {
     // ------ authentication logic
 
     private static final Set<Code> OTHER_AUTH_FAILURE_CODES = ImmutableSet.of(
-            Code.INVALID_ARGUMENT, Code.FAILED_PRECONDITION, Code.PERMISSION_DENIED, Code.UNKNOWN);
+            Code.INVALID_ARGUMENT, Code.FAILED_PRECONDITION, Code.PERMISSION_DENIED, Code.INTERNAL, Code.UNKNOWN);
```

Comment:
There was a problem with incorrect auth case identification. Had to add Code.INTERNAL to OTHER_AUTH_FAILURE_CODES among existing.